### PR TITLE
editorial pass of §5 up to 5.3.2

### PR DESCRIPTION
**Pull request by thomas-fossati**
* various reformatting
* edits for clarity & conciseness
* added missing "Evidence Appraisal" subsection
* removed "group" from the glossary because it wasn't used

### CODE REVIEW

**Comment by thomas-fossati** [comment]
While doing the editorial pass, I realised I didn't get the intended scope of §5.1 "Verifier Abstraction", at least given its current contents which can be categorised into two buckets:
1. a restate of the last paragraph of §5
2. many useful terms

The former can probably be dropped (or merged with what we already have in §5), while the latter needs to move to the terminology section (§1.1)

I guess we might want to put a picture similar to https://github.com/ietf-rats-wg/draft-ietf-rats-corim/pull/196 nearby.

File: draft-ietf-rats-corim.md [hunk 1]
```diff
@@ -1388,137 +1389,106 @@ phase can be amortised across multiple appraisals.
...
 Other selection criteria MAY be applied.
-
-For example, if the Evidence format is known in advance, CoRIMs using a
-profile that is not understood by a Verifier can be readily discarded.
+For example, if the Evidence format is known in advance, CoRIMs using a profile that is not understood by a Verifier can be readily discarded.
 
 The selection process MUST yield at least one usable tag.
 
-### CoBOM Extraction
+Later stages will further select the CoRIMs appropriate to the Evidence Appraisal stage. TBC(tho)
```
Comment (thomas-fossati) [inline draft-ietf-rats-corim.md:1438]:
```suggestion
Later stages will further select the CoRIMs appropriate to the Evidence Appraisal stage.
```

**Comment by thomas-fossati** [comment]
Another observation I have is that this section should be the entry point of the document.  From here we should have references to the sections describing the data structures used.

File: draft-ietf-rats-corim.md [hunk 2]
```diff
@@ -1388,137 +1389,106 @@ phase can be amortised across multiple appraisals.
...
-This concludes the initialisation phase.
-
-## Evidence Collection
+## Evidence Collection {#sec-ev-coll}
 
-In the Evidence collection phase the Verifier communicates with attesters to
-collect Evidence.
+During the Evidence collection phase, the Verifier communicates with attesters to gather Evidence.
+The first part of this phase does not involve any cryptographic validation.
+This means that Verifiers can use untrusted code for their initial Evidence collection.
```
Comment (nedmsmith) [inline draft-ietf-rats-corim.md:1475]:
```suggestion
This means that Verifiers MAY use untrusted code for their initial Evidence collection.
```

File: draft-ietf-rats-corim.md [hunk 3]
```diff
@@ -1388,137 +1389,106 @@ phase can be amortised across multiple appraisals.
...
-certificate is looked up -- e.g., based on linking information obtained from
-the DeviceID certificate (see Section 9.2.1 of {{DICE.Layer}}) --
-in the Appraisal Context.  If found, then usual X.509 certificate validation
-is performed.
-In PSA, the verification public key is looked up in the appraisal context using
-the `ueid` claim found in the PSA claims-set (see {{Section 4.2.1 of -psa-token}}).
+If Evidence is cryptographically signed, its validation is the first step in Appraisal.
+
+The way cryptographic signature validation works depends on the specific Evidence collection protocol being used.
+For instance, in DICE, a proof of liveness is carried out on the final key in the certificate chain.
```
Comment (nedmsmith) [inline draft-ietf-rats-corim.md:1485]:
```suggestion
For instance, in DICE, a proof of liveness is carried out on the final key in the certificate chain (a.k.a., the alias certificate).
```

File: draft-ietf-rats-corim.md [hunk 4]
```diff
@@ -1388,137 +1389,106 @@ phase can be amortised across multiple appraisals.
...
+If this is successful, a suitable certification path is looked up in the Appraisal Context, based on linking information obtained from the DeviceID certificate (see Section 9.2.1 of {{DICE.Layer}}).
+If a trusted root certificate is found, the usual X.509 certificate validation is performed.
+On the other hand, in PSA, the verification public key is looked up in the appraisal context using the `ueid` claim found in the PSA claims-set (see {{Section 4.2.1 of -psa-token}}).
```
Comment (nedmsmith) [inline draft-ietf-rats-corim.md:1488]: Should this text be moved to the PSA profile?

**Review by nedmsmith — COMMENTED** [review]
see comments inline

File: draft-ietf-rats-corim.md (hunk 2 above)
Comment (thomas-fossati) [reply draft-ietf-rats-corim.md:1475]: RFC2119 terms are for interop requirements.  This one is an implementation choice instead.

File: draft-ietf-rats-corim.md (hunk 4 above)
Comment (thomas-fossati) [reply draft-ietf-rats-corim.md:1488]: I am not against dropping this.  However, it provides a concrete example of how different profiles may plug in their logic.

Comment (nedmsmith) [reply draft-ietf-rats-corim.md:1488]: Lines 1485-1487 appear to apply to a DICE example. Line 1488 doesn't begin with enough descriptive text to highlight it as a second example, vs. an extension to a DICE example. As a second example, it needs more development to make it understandable. I didn't want to spend too much time in this document reviewing content that applies to a different audience. Possibly, a reference to the psa document is sufficient for an interested reader to go there to obtain the well-described example?

**Review by nedmsmith — APPROVED** [review]
Conditionally approved given line 1488 is reworded to distinguish it as a second example or possibly remove it for now.

File: draft-ietf-rats-corim.md [hunk 5]
```diff
@@ -1388,137 +1389,106 @@ phase can be amortised across multiple appraisals.
...
-is performed.
-In PSA, the verification public key is looked up in the appraisal context using
-the `ueid` claim found in the PSA claims-set (see {{Section 4.2.1 of -psa-token}}).
+If Evidence is cryptographically signed, its validation is the first step in Appraisal.
+
+The way cryptographic signature validation works depends on the specific Evidence collection protocol being used.
+For instance, in DICE, a proof of liveness is carried out on the final key in the certificate chain (a.k.a., the alias certificate).
+If this is successful, a suitable certification path is looked up in the Appraisal Context, based on linking information obtained from the DeviceID certificate (see Section 9.2.1 of {{DICE.Layer}}).
+If a trusted root certificate is found, the usual X.509 certificate validation is performed.
+On the other hand, in PSA, the verification public key is looked up in the appraisal context using the `ueid` claim found in the PSA claims-set (see {{Section 4.2.1 of -psa-token}}).
```
Comment (thomas-fossati) [inline draft-ietf-rats-corim.md:1488]:
```suggestion
As a second example, in PSA {{-psa-token}} the verification public key is looked up in the appraisal context using the `ueid` claim found in the PSA claims-set.
```

**Comment by thomas-fossati** [comment]
> Conditionally approved given line 1488 is reworded to distinguish it as a second example or possibly remove it for now.

See if [541d4c3](https://github.com/ietf-rats-wg/draft-ietf-rats-corim/pull/196/commits/541d4c3118230a2ef67646cbbcc0a61bb579103e) works for you.

**Review by nedmsmith — APPROVED** [review]
LGTM

**Pull request merged**